### PR TITLE
feature/SEM-69 Bibliographies Top Elements

### PR DESCRIPTION
**Pull request by VictoriaCk55**
image

### CODE REVIEW

**Comment by VictoriaCk55** [comment]
https://ditmar.github.io/backoffice-openscience/branch-SEM-69/?path=/story/molecules-bibliographyelements--default

File: src/ui/components/molecules/BibliographyElements/BibliographyElements.module.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+.config {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 0.8rem;
+
+    >button {
+        border: none !important;
```
Comment (Ditmar) [inline src/ui/components/molecules/BibliographyElements/BibliographyElements.module.scss:34]: please remove the use of the !important

File: src/ui/components/molecules/BibliographyElements/BibliographyElements.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          size="small"
+          onClick={setFilter}
+          variant="secondary"
+        />
+
+        <ConfigureViewButton
+          icon={settings}
+          size="small"
+          onClick={() => {
+            console.log('clicked');
```
Comment (Ditmar) [inline src/ui/components/molecules/BibliographyElements/BibliographyElements.tsx:26]: remove the console log

**Review by Ditmar — CHANGES_REQUESTED** [review]
read the comments

**Review by Ditmar — APPROVED** [review]

**Pull request merged**